PowerExponentiation/powerExpo.cpp

```diff
@@ -0,0 +1,40 @@
+/****************************************************
+
+@author: vichitr
+Compiled On: 9th Oct 2019
+
+****************************************************/
+#include<bits/stdc++.h>
+#define MAX 9223372036854775807
+#define endl "\n"
+#define ll long long 
+// #define int unsigned long long 
+
+#define double long double
+#define pb push_back
+#define pf pop_front
+#define mp make_pair
+#define pii pair<int, int>
+
+#define loop(i,n) for(int i=0;i<n;i++)
+#define loops(i,s,n) for(int i=s;i<=n;i++)
+#define fast ios::sync_with_stdio(0); cin.tie(NULL); cout.tie(NULL)
```

Comment:
Hi, @vichitr Thanks for the PR, but there is a compilation error while running this code.  These definitions are not being used in the code also there is no main() function to run pwr function.

Reply:
Hey! I have added main now. Initially it was just a function.